indy_node/utils/node_control_tool.py

```diff
@@ -272,7 +273,8 @@ def _run_shell_script(cls, command, timeout):
                               timeout=timeout)
 
     def start(self):
-        self._hold_packages()
+        if 'ubuntu' in platform.platform().lower():
```

Comment:
why 'ubuntu' only, e.g. such a conditional will fail on debian where apt is also available

Reply:
So checking of apt-mark executable is needed to be more generic